lib/api/errors/not_found.rb

```diff
@@ -31,7 +31,7 @@ module API
   module Errors
     class NotFound < ErrorBase
       def initialize(message)
-        super 404, message + '.'
+        super 404, message
```

Comment:
@ulferts So you propose to use the localized string *instead* of ``message`` here or in just in case the ``message`` is empty?